backend/blender_script/main.py

```diff
@@ -135,11 +129,20 @@ def startup(SPHR, SPHL, frame, PD):
 
 if __name__ == "__main__":
     print(sys.argv)
-    if len(sys.argv) < 5:
+    if len(sys.argv) < 10:
         print('shouldnt be here')
-        startup(0.5, 0.5, "aviator", 64)
+        startup(0.5, 0.5, 0, 0, 0, 0, 1.5, "aviator", 64)
     else:
-        file_path = "backend/models/"+str(sys.argv[3])+"_inset.blend"
+        file_path = "backend/models/"+str(sys.argv[8])+"_inset.blend"
         bpy.ops.wm.read_factory_settings(use_empty=True)
         bpy.ops.wm.open_mainfile(filepath=file_path)
-        startup(float(sys.argv[1]), float(sys.argv[2]), sys.argv[3], float(sys.argv[4]))
\ No newline at end of file
+        startup(float(sys.argv[1]), #SPHR
```

Comment:
Same thing here. I think it's more elegant to read all the sys.argv's into a Prescription class, then pass that into startup